docs/modules/ROOT/pages/how-tos/cloudscale/enable-loadbalancer-service.adoc

```diff
@@ -0,0 +1,365 @@
+= Enable `LoadBalancer` serivces on cloudscale.ch
+
+[abstract]
+--
+Steps to enable `LoadBalancer` serivces on an OpenShift 4 cluster on https://cloudscale.ch[cloudscale.ch].
+--
+
+[NOTE]
+--
+- In the future, this procedure will be mostly automated
+--
+
+== Starting situation
+
+* You already have an OpenShift 4 cluster on cloudscale.ch
+* You have admin-level access to the cluster
+* The cluster is using https://cilium.io[Cilium] as the network plugin
+* You want to enable LoadBalancer services
+
+== Prerequisites
+
+* `ssh`
+* `curl`
+* `kubectl`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
+* `whois`
+
+== Prepare IP range
+
+. Get a suitable public IP range.
+You can find all IP ranges owned by VSHN
++
+[source,console]
+----
+whois -i mnt-by MNT-VSHN -T inetnum -r
+----
+Please coordinate with other VSHNeers before using an IP range.
```

Comment:
Who is "other VSHNeers"? We might need to be more specific here.

Reply:
The thing is: I'm not sure. It doesn't seem like there is really someone that feels responsible.

Reply:
```suggestion
Please coordinate with VSHN Corporate IT before using an IP range.
```